student_div.c

```diff
@@ -1,19 +1,20 @@
 #include<stdio.h>
-Int main()
+int main()
 {
-    Int m1,m2,m3,m4,m5,per;
-    Printf("enter marks in five subjects");
-    Scanf("%d%d%d%d%d",&m1,&m2,&m3,&m4,&m5);
+    int m1,m2,m3,m4,m5;
+    float per;
+    printf("enter marks in five subjects: ");
+    scanf("%d%d%d%d%d",&m1,&m2,&m3,&m4,&m5);
 
-     Per=(m1+m2+m3+m4+m5)/500*100;
+    per=(m1+m2+m3+m4+m5)*100/500; 
+    printf("Percentage: %f\n", per);
+    if(per>=60)
+          printf("first division\n");
      
-    If(per>=60)
-          Printf("first division\n");
-     
-    If((per>=50)&&(per<50))
-          Printf("second division\n");
+    else if((per>=50)&&(per<60))
+          printf("second division\n");
 
-    If(per<40)
-          Printf("fail\n");
+    else if(per<40)
```

Comment:
At last, it should be else.  What do you think?

Reply:
Sir, according to the original logic of the program (as written by Komal), students below 40% should only fail but if we remove the condition then even the students below 50% will fail.

Reply:
Sir, I updated the program, kindly let me know if any other change is required.

Reply:
yes, its fine - you can merge. I it cann't be like program is exiting improperly if per is in between 40 and 50. So, let students below 50 fail. 
Program looks good to me. Thanks @komal7141 and @Anjali-1801  !! Nice to see the collaboration.

Reply:
Yes sir, I changed the condition and added a third division as well, now all the students below 40% will fail. I'll merge the PR.